src/blocks/AppsProcessor.cpp

```diff
@@ -69,21 +69,23 @@ void AppsProcessor::evaluate(VcuParameters* params, AppsProcessorInput* input,
     }
     else {
         output->ok = true;
-        float preconv = (app1Perc + app2Perc) / 2; // TODO average
+        float preconv = (app1Perc + app2Perc) / 2;
+        // calculate slope
+        float slope = 1/(1 - params->appsDeadZonePct);
```

Comment:
i think you would subtract appsDeadZonePct twice (multiply it by 2 before subtracting) because it's on both ends